manim/animation/rotation.py

```diff
@@ -17,6 +17,19 @@
 
 
 class Rotating(Animation):
+    """A mobject revolves about a certain point.
+
+    Examples
+    --------
+    .. manim:: UsingRotating
+
+        class UsingRotating(Scene):
+            def construct(self):
+                self.add(Dot(ORIGIN))
+                self.play(Rotating(Square(side_length=0.5).shift(UP*2), about_point=ORIGIN), run_time=5)
```

Comment:
This example has a few too many characters in one line, consider running `python -m black` on a a file which has this example so that it can fit in the window:
<img width="641" alt="image" src="https://user-images.githubusercontent.com/70682032/149073998-cf1764b1-ca19-4240-83ce-244d51f7a477.png">

Reply:
yeah bro, sure